### PR TITLE
Revert "Revert "[DESIGN2-104] Dropdowns - Add thick and thin label type style…"

### PR DESCRIPTION
**Pull request by levadadenys**
### Revert "Revert "[DESIGN2-104] Dropdowns - Add thick and thin label type style…"

* reverts https://github.com/code-dot-org/code-dot-org/pull/58017
* fixes missing `marginBottom` for `SectionProgress` sceen

Before (production):
Знімок екрана 2024-04-16 о 14 11 06

After:
Знімок екрана 2024-04-16 о 14 54 02

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.story.tsx [hunk 1]
```diff
@@ -43,41 +43,22 @@ const MultipleTemplate: StoryFn<{
...
-                  [componentArg.name]: e.target.value,
-                });
-                componentArg.onChange(e);
-              }}
-            />
-          )
-        )}
+        {args.components?.map(componentArg => (
+          <SimpleDropdown
+            key={`${componentArg.name}`}
```
Comment (thomasoniii) [inline apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.story.tsx]: can this just be keyed as `key={componentArg.name}`? Or is it trying to force it as a string?

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

**Review by lfryemason — APPROVED** [review]

**Review by lfryemason — CHANGES_REQUESTED** [review]
Forgot to check one thing in original approval. This also modifies the margin on the new progress view. Can we also add a margin there before merging?
image

**Review by lfryemason — APPROVED** [review]
Sorry been a long day. I forgot that I already fixed the other instance in [THIS PR](https://github.com/code-dot-org/code-dot-org/pull/57993/files). Should be ok to merge this one.

**Pull request merged**